test/lib/ansible_test/_util/controller/sanity/pylint/plugins/deprecated.py

```diff
@@ -218,8 +229,9 @@ def visit_call(self, node):
         date = None
         collection_name = None
         try:
-            if (node.func.attrname == 'deprecated' and 'display' in _get_expr_name(node) or
-                    node.func.attrname == 'deprecate' and _get_expr_name(node)):
+            funcname = _get_func_name(node)
+            if (funcname == 'deprecated' and 'display' in _get_expr_name(node) or
```

Comment:
@sivel I realize this was already `'display' in _get_expr_name(node)` originally, but do you know why it's not `'display' == _get_expr_name(node)` instead?

Reply:
It looks like I've answered my own question. It seems `_display` is sometimes used instead of `display`:

https://github.com/ansible/ansible/blob/d5f35783695d94937a0ffca1dc1843df06e5680f/lib/ansible/plugins/action/__init__.py#L876-L877